demisto_sdk/commands/validate/initializer.py

```diff
@@ -506,20 +505,15 @@ def git_paths_to_basecontent_set(
         basecontent_with_path_set: Set[BaseContent] = set()
         invalid_content_items: Set[Path] = set()
         non_content_items: Set[Path] = set()
-        git_util = GitUtil.from_content_path()
-        current_git_sha = git_util.get_current_git_branch_or_hash()
         for file_path, git_status in statuses_dict.items():
             if git_status == GitStatuses.DELETED:
                 continue
             try:
                 old_path = file_path
                 if isinstance(file_path, tuple):
                     file_path, old_path = file_path
-                obj = BaseContent.from_path(
-                    file_path, git_sha=current_git_sha, raise_on_exception=True
-                )
+                obj = BaseContent.from_path(file_path, raise_on_exception=True)
                 if obj:
-                    obj.git_sha = current_git_sha
```

Comment:
why are we removing this part as well?
I thought we do want to keep some sort of a way to reach out to current git_sha

Reply:
Any news on this? Eagerly awaiting this PR to be merged...

Reply:
@YuvHayun You are correct.
I removed the `current_git_sha` from `BaseContent.from_path` creation to prevent the parser from attempting to rely on a non-existent remote branch.
While also adding back the `obj.git_sha = current_git_sha` to ensure that the `current_git_sha` is  still available as the `self.git_sha` attribute  in the `BaseContent` object for relevant validations.

Reply:
@tlium Thank you for your feedback, this PR is currently in review.